06-building-a-program-start-to-finish/setting-up-your-repository/output/README.md

```diff
@@ -0,0 +1,8 @@
+#### Output
+- **Using**: GitHub
+- **Deliver**: the repository for your proposed project and at least one `Pull Request`
+- **With**: *independently*
+
+####Link to GitHub Repo
+- [ andela-uibrahim / invertedIndex](https://github.com/andela-uibrahim/inverted-index/pull/2)
```

Comment:
I believe I've addressed this issue of ReadMe files being empty.

Kindly address.